interfaces/dbus/backend.go

```diff
@@ -88,6 +105,33 @@ func setupDbusServiceForUserd(snapInfo *snap.Info) error {
 	return nil
 }
 
+func setupHostDBusConf(snapInfo *snap.Info) error {
+	sessionContent, systemContent, err := wrappers.DeriveSnapdDBusConfig(snapInfo)
```

Comment:
Should we attempt to remove such files or copy / backup originals in case we fail or we get removed?

Reply:
The goal of the existing code was to give a functional snapd if the user upgrades via re-exec before the matching distro package version of snapd is released.  The new files would eventually be adopted by the new distro snapd when the user upgrades.

It's not clear there is a reasonable way to support going backwards, or if that is desirable.  I'd originally left this code out of the PR, but was told we still want this feature even if it is not perfect.